docs/getting-started-tutorial/8-customizable-usernames.adoc

```diff
@@ -0,0 +1,69 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# Customizable Usernames
+
+Let's deal with the next use case:
+
+ - As a chatter, I want to change what name other users see for me when I post
+   a message.
+
+What we really want is a text box on the client that will let us edit the name.
+We'll add it to the top of our chat area in `chat.html`:
+
+```html
+...
+  <section id="chat">
+    <header>
+      Posting as:
+      <input data-lift="Chat.nameField" class="poster-name">
+    </header>
+...
+```
+
+The ideal way for this to work would be for you to be able to change the value
+of the field, and have it save. We can do exactly that using Lift's `ajaxText`
```

Comment:
Not sure about the comma after `field`. Might also be worthwhile to make mention that the save occurs on blur?